gradle.properties

```diff
@@ -0,0 +1,11 @@
+# default Docker image to use for launching applications
+# keep these properties in-sync with the DockerAppsRunner.DEFAULT_DOCKER_IMAGE_ID Java property
+# dockerGcrHostname/dockerGcrProject/dockerImageName:dockerImageTag
```

Comment:
nit: should be removed?

Reply:
Do you mean line 3? No, that's showing how to concatenate the various properties into the full image path.

Reply:
Yep, that wasn't clear to me, sorry.

Reply:
Added comment about concatenation.